samples/CheckoutSdk.SampleApp/CustomCkoHttpClientFactory.cs

```diff
@@ -0,0 +1,19 @@
+using System.Net.Http;
+
+namespace Checkout.SampleApp
+{
+    public class CustomCkoHttpClientFactory : IHttpClientFactory
+    {
+        private readonly System.Net.Http.IHttpClientFactory _msHttpClientFactory;
```

Comment:
I guess this could be simplified ~`System.Net.Http.IHttpClientFactory`~ -> `IHttpClientFactory`

Reply:
@philippe-leonhardt-cko unfortunately not as it will conflict with the Checkout SDK `IHttpClientFactory`